### PR TITLE
Addressing ill formed links.

### PR DESCRIPTION
**Pull request by gsaluja9**

### CODE REVIEW

File: aperturedb/Connector.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ class Connector(object):
     """
     **Class to facilitate connections with an instance of aperturedb**
 
-    It lets the client execute any query as per the `Native Query specs </category/aperturedb-query-language>`_
+    It lets the client execute any query as per the [Native Query specs](/query_language/Overview/API%20Description)
```
Comment (vishakha041) [inline aperturedb/Connector.py:80]:
```suggestion
    It lets the client execute any JSON query based on the [ApertureDB query language specification](/query_language/Overview/API%20Description)
```

File: aperturedb/PytorchData.py [hunk 2]
```diff
@@ -4,6 +4,21 @@
 
 
 class PytorchData(Subscriptable):
```
Comment (vishakha041) [inline aperturedb/PytorchData.py:6]:
```suggestion
class PyTorchData(Subscriptable):
```

File: aperturedb/PytorchData.py [hunk 3]
```diff
@@ -4,6 +4,21 @@
...
+    """
+    **Class to wrap around a Dataset retrieved from torchvision.datasets**
```
Comment (vishakha041) [inline aperturedb/PytorchData.py:8]: Do we have a link to refer here for torch vision?

File: aperturedb/PytorchData.py [hunk 4]
```diff
@@ -4,6 +4,21 @@
...
+
+    The dataset in this case can be iterated over to yield atomic records.
```
Comment (vishakha041) [inline aperturedb/PytorchData.py:10]: What does atomic records here mean?

**Review by vishakha041 — CHANGES_REQUESTED** [review]
A few changes and this looks good

**Review by JoshStoddard — COMMENTED** [review]
👍 LGTM

File: aperturedb/PytorchData.py (hunk 3 above)
Comment (gsaluja9) [reply aperturedb/PytorchData.py:8]: will add

File: aperturedb/PytorchData.py (hunk 4 above)
Comment (gsaluja9) [reply aperturedb/PytorchData.py:10]:
```suggestion
    The dataset in this case can be iterated over.
```

**Pull request merged**